tests/Cbor/DecodeTests.elm

```diff
@@ -374,33 +374,42 @@ suite =
                      in
                      any |> andThen fromCborItem
                     )
-                    (Just <| CborMap [ ( CborInt 1, CborInt 1 ), ( CborInt 2, CborInt 2 ) ])
+                    (Just <| CborMap [ ( CborInt32 1, CborInt32 1 ), ( CborInt32 2, CborInt32 2 ) ])
             ]
         , describe "any / raw"
             [ hex [ 0x00 ]
```

Comment:
Could we add a couple of tests here covering the case with `any` and integers larger than 2^53 ?